test/module/irohad/ametsuchi/CMakeLists.txt

```diff
@@ -39,6 +39,18 @@ target_link_libraries(kv_storage_test
     libs_common
     )
 
+addtest(storage_test storage_test.cpp)
+target_link_libraries(storage_test
+    ametsuchi
+    libs_common
+    )
+
+addtest(postgres_options_test postgres_options_test.cpp)
+target_link_libraries(postgres_options_test
+    ametsuchi
```

Comment:
Looks like tests of parsing shouldn't depend on storage, isn't it?

Reply:
postres options is part of ametsuchi target